### PR TITLE
Update man page

### PR DESCRIPTION
**Pull request by williamcroberts**

### CODE REVIEW

**Comment by williamcroberts** [comment]
> I didn't understand how the two commands work. Perhaps that should be explained in a bit more detail.

Updated to be more clear.

File: man/tss2-tcti-cmd.7.in [hunk 1]
```diff
@@ -12,3 +12,20 @@ with the interface and protocol exposed by a sub-process that can receive and tr
 raw TPM2 command and response buffers. The interface exposed by this library is defined
 in the \*(lqTSS System Level API and TPM Command Transmission Interface
 Specification\*(rq specification.
+.SH EXAMPLES
+.sp
+For example, if you wanted to use the tpm2_send(1) command as the sub-process to
+send data to and from the TPM, one could do so like this:
+.nf
+tpm2_getrandom -T "cmd:tpm2_send -s" --hex 4"
```
Comment (JuergenReppSIT) [inline man/tss2-tcti-cmd.7.in:20]: I think the " after --hex 4 should be removed.

**Review by JuergenReppSIT — CHANGES_REQUESTED** [review]

File: man/tss2-tcti-cmd.7.in (hunk 1 above)
Comment (williamcroberts) [reply man/tss2-tcti-cmd.7.in:20]: Ahh yep

File: man/tss2-tcti-cmd.7.in [hunk 2]
```diff
@@ -3,12 +3,29 @@
...
+.fi
+.sp
+A more useful example would be connecting to a remote machine using ssh and
+interacting with the remote machine's TPM as if it were local. This uses
+tpm2_send(1) on a remote machine and uses ssh as the sub-process to create
+the tunnel to the remote machine. If encrypted sessions are used, the
+remote machine is blinded to the TPM data and cannot interpose on that
+traffic.
+.nf
+tpm2_getrandom -T "cmd:ssh remotehost tpm2_send -s --hex 4"
```
Comment (williamcroberts) [inline man/tss2-tcti-cmd.7.in:30]: The " needs to move here to the TCTI string....

**Review by JuergenReppSIT — APPROVED** [review]

**Pull request merged**